### PR TITLE
9980 jar progress description intermediate branch to test

### PR DESCRIPTION
**Pull request by rgoshen**
This adds in an integration test for Judge Activity Report Progress Description table and addresses PO feedback.

Now in the Progress Description table, it will show cases with even codes of ODD, DEC, OAD, SDEC if they are unserved or in draft status.

### CODE REVIEW

File: shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts [hunk 1]
```diff
@@ -70,7 +80,7 @@ export const getCasesByStatusAndByJudgeInteractor = async (
       statuses,
     });
 
-  const rawCaseRecords = await Promise.all(
+  const rawCaseRecords: any = await Promise.all(
```
Comment (TomElliottFlexion) [inline shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts]: Could these be type RawCase?

File: web-client/src/presenter/computeds/JudgeActivityReport/judgeActivityReportHelper.ts [hunk 2]
```diff
@@ -1,7 +1,23 @@
 import { state } from 'cerebral';
 import { sum, sumBy } from 'lodash';
 
-export const judgeActivityReportHelper = (get, applicationContext) => {
+interface IJudgeActivityReportHelper {
```
Comment (TomElliottFlexion) [inline web-client/src/presenter/computeds/JudgeActivityReport/judgeActivityReportHelper.ts:4]: Nice

**Review by TomElliottFlexion — APPROVED** [review]

File: shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts (hunk 1 above)
Comment (rgoshen) [reply shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.ts]: Done! I changed it to RawCase[] type.  Thanks

File: web-client/src/presenter/computeds/JudgeActivityReport/judgeActivityReportHelper.ts (hunk 2 above)
Comment (rgoshen) [reply web-client/src/presenter/computeds/JudgeActivityReport/judgeActivityReportHelper.ts:4]: I will pass it on to Tim

**Pull request merged**